src/h3/to_multipoly.pyx

```diff
@@ -0,0 +1,88 @@
+cimport h3lib
+from h3lib cimport H3int
+from .util cimport check_addr
+
+
+# need to define `coord2geo` and `mercator` here due to Cython bug:
+# https://github.com/cython/cython/issues/2745
+cdef (double, double) mercator(double lat, double lng):
```

Comment:
I'd suggest renaming to something like `constrainCoords` - this doesn't seem to have anything to do with the Mercator projection.

Reply:
Is this still needed? I'm pretty sure we don't do it in JS any more - we added it into the C lib, so I think all values should be within the expected range.

Reply:
Correct, C output is still radians.

Reply:
got it!